src/tabletraits.jl

```diff
@@ -0,0 +1,42 @@
+Tables.isrowtable(::Type{<:DESolution}) = true
+Tables.columns(x::DESolution) = Tables.columntable(Tables.rows(x))
```

Comment:
Did we have this definition previously? At first glance, it doesn't seem necessary (the fallback in Tables.jl should do the same thing)

Reply:
Otherwise, all the Tables.jl stuff here looks great